### PR TITLE
Update Docker inspect command reference doc to match Docker 1.11

### PR DESCRIPTION
**Pull request by charleswhchan**
I notice the information is slightly outdated, so a quickly update.
- Also touch up capitalization in the headings.

Signed-off-by: Charles Chan charleswhchan@users.noreply.github.com

### CODE REVIEW

File: docs/reference/commandline/inspect.md [hunk 1]
```diff
@@ -47,14 +46,14 @@ straightforward manner.
...
-**List All Port Bindings:**
+**List all port bindings:**
 
 One can loop over arrays and maps in the results to produce simple text
 output:
 
     $ docker inspect --format='{{range $p, $conf := .NetworkSettings.Ports}} {{$p}} -> {{(index $conf 0).HostPort}} {{end}}' $INSTANCE_ID
 
-**Find a Specific Port Mapping:**
+**Find a apecific port mapping:**
```
Comment (thaJeztah) [inline docs/reference/commandline/inspect.md]: s/apecific/specific/

**Comment by thaJeztah** [comment]
Thanks @charleswhchan, saw one typo, but looking good otherwise

File: docs/reference/commandline/inspect.md (hunk 1 above)
Comment (charleswhchan) [inline docs/reference/commandline/inspect.md]: Fixed. :sweat_smile:

**Comment by charleswhchan** [comment]
Fixed typo.

**Comment by thaJeztah** [comment]
Oh,  sorry @charleswhchan, looks like this needs a rebase now, because https://github.com/docker/docker/commit/9acf97b72a4d5ff7b1bcad36fb19b53775f01596 was merged

**Comment by SvenDowideit** [comment]
LGTM assuming rebase.

**Comment by thaJeztah** [comment]
carried in https://github.com/docker/docker/pull/23717

**Pull request closed without merging**